packages/redis/src/__test__/redisClient.integration.test.ts

```diff
@@ -55,4 +56,17 @@ describe("redis integration test", () => {
         const outputPayload = await client.getObject(span, TestClass, aKey);
         expect(outputPayload).toMatchObject(msg.payload);
     });
+
+    it("successfully xadds a value into a stream and returns the id", async () => {
```

Comment:
can you add an end-to-end test here? for instance create a CC Application with a StaticInputSource and then use the RedisStreamOutputSink ... run the application and use the redis client directly to check that the msgs made all the way.

fyi: you can make CI run all the integration tests if you include "[full ci]" in your git commit message ... by default we only run unit tests